include/crow/http_server.h

```diff
@@ -207,33 +207,36 @@ namespace crow
 
         void do_accept()
         {
-            uint16_t service_idx = pick_io_service_idx();
-            asio::io_service& is = *io_service_pool_[service_idx];
-            task_queue_length_pool_[service_idx]++;
-            CROW_LOG_DEBUG << &is << " {" << service_idx << "} queue length: " << task_queue_length_pool_[service_idx];
-
-            auto p = new Connection<Adaptor, Handler, Middlewares...>(
-              is, handler_, server_name_, middlewares_,
-              get_cached_date_str_pool_[service_idx], *task_timer_pool_[service_idx], adaptor_ctx_, task_queue_length_pool_[service_idx]);
-
-            acceptor_.async_accept(
-              p->socket(),
-              [this, p, &is, service_idx](boost::system::error_code ec) {
-                  if (!ec)
-                  {
-                      is.post(
-                        [p] {
-                            p->start();
-                        });
-                  }
-                  else
-                  {
-                      task_queue_length_pool_[service_idx]--;
-                      CROW_LOG_DEBUG << &is << " {" << service_idx << "} queue length: " << task_queue_length_pool_[service_idx];
-                      delete p;
-                  }
-                  do_accept();
-              });
+            if (!should_close_)
```

Comment:
Maybe change name to `working` and invert states? Then it should be easier to understand (no need to invert) ;)

Reply:
what about something like `shutting_down_`? Since the boolean is only true while Crow is closing

Reply:
Hmm, it need to be inverted, but is IMHO more readable than the current version ;)

Reply:
I believe I already inverted the states in the past commit, I'll push the name change now :)